docs/about/architecture.md

```diff
@@ -162,3 +162,104 @@ offer the fastest developer experience.
 - Session persistence (sticky sessions) can be disabled as _coderd_ instances
   are stateless.
 - WebSocket and long-lived connections must be supported.
+
+### Multi-cloud architecture
+
+By distributing Coder instances across different cloud providers, organizations
```

Comment:
> By distributing Coder instances across different cloud providers

The diagram shows Coder instances only in GCP, with only workspaces distributed across clouds.

```suggestion
By distributing Coder workspaces across different cloud providers, organizations
```

Are multi-cloud control planes recommended? This would necessitate cross-cloud database master-master replication, which seems unnecessary, and should probably be called out specifically.

Reply:
You're right! Multi-cloud control planes should not be recommended. I adjusted `s/instances/workspaces/` and added an extra note about control planes.